### PR TITLE
Remove `since` from options and languages

### PR DESCRIPTION
**Pull request by fisker**
## Description

Since v2.0.0, we removed `version` option from `getSupportInfo` #7620, `since` became useless, let's remove it.

## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by fisker** [comment]
It can be useful, but if we do https://github.com/prettier/prettier/issues/9879#issuecomment-895183665, we won't need that. Even if we want run multiple version in on page, we can still hardcode there.

**Comment by thorn0** [comment]
I forgot that every version can tell us itself what it supports. So let's proceed.

**Comment by fisker** [comment]
Added old values to playground, so old link won't be broken https://github.com/prettier/prettier/pull/13699/commits/1ba50c73b5465b4fb3d52c67c9ba244e0c49a518

File: src/main/support.js [hunk 1]
```diff
@@ -16,49 +12,30 @@ import currentVersion from "./version.evaluate.cjs";
...
-      if (Array.isArray(option.default)) {
-        option.default =
-          option.default.length === 1
-            ? option.default[0].value
-            : option.default
-                .filter(filterSince)
-                .sort((info1, info2) =>
-                  semverCompare(info2.since, info1.since)
-                )[0].value;
-      }
```
Comment (fisker) [inline src/main/support.js:57]: Maybe I should keep this array thing, otherwise it will be a breaking change, plugin may use array as default value.

Comment (fisker) [reply src/main/support.js]: Deprecated array, introduced function for non-primitive values, it will be safer (to avoid mutate).

Comment (thorn0) [reply src/main/support.js]: Shouldn't these things be JSON-serializable?

**Comment by fisker** [comment]
> Also this logic was used to hide unreleased things on the playground.

I know that, but why? Only PR's playground will show unreleased, it should be shown, let people with it.

**Comment by fisker** [comment]
And we already use `showUnreleased: true,`
https://github.com/prettier/prettier/blob/14c58aafd899f9b0b6ab7f7b9713b8f9eed36c09/website/static/worker.js#L82

File: src/main/support.js (hunk 1 above)
Comment (fisker) [reply src/main/support.js]: Make sense, so I'll just put `[{value: []}]` back.

Comment (fisker) [reply src/main/support.js]: Wait, we get the support info from `getSupportInfo` function, so it will return `[]`, then it's serializable.

Comment (fisker) [reply src/main/support.js]: The option self is not serializable, there are functions for `exception`.

**Comment by thorn0** [comment]
> And we already use `showUnreleased: true,`
> 
> https://github.com/prettier/prettier/blob/14c58aafd899f9b0b6ab7f7b9713b8f9eed36c09/website/static/worker.js#L82

Doesn't this PR remove support for `showUnreleased`?

**Comment by fisker** [comment]
`showUnreleased` checks `since`, there is no `since` to check now.

**Comment by fisker** [comment]
I'm we already show unreleased before this PR, so 

> Also this logic was used to hide unreleased things on the playground.

We are not using.

**Comment by fisker** [comment]
Don't overthink this, nobody is using it, even I thought `prettier-vscode` may use it to support different version of prettier, but they are not.

File: src/main/support.js (hunk 1 above)
Comment (fisker) [reply src/main/support.js]: Let's add a new `defaultValue` field?

Comment (thorn0) [reply src/main/support.js]: How will it be different from `default`?

Comment (fisker) [reply src/main/support.js]: When they are array, `defaultValue` use the value directly, `default` use `default[0].value`, so we don't need ask plugin to update.

Comment (fisker) [reply src/main/support.js]: So I can use `defaultValue: []` instead of `default: [{value: []}]`.

Comment (fisker) [reply src/main/support.js]: Forget it, switched back to  `default: [{value: []}]`

**Review by thorn0 — APPROVED** [review]

**Review by sosukesuzuki — APPROVED** [review]
Good catch

**Pull request merged**